src/Search_Replace_Command.php

```diff
@@ -367,6 +368,7 @@ public function __invoke( $args, $assoc_args ) {
 				if ( ! $php_only && ! $this->regex ) {
 					$col_sql          = self::esc_sql_ident( $col );
 					$wpdb->last_error = '';
+					// phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared -- escaped through self::esc_sql_ident
 					$serial_row       = $wpdb->get_row( "SELECT * FROM $table_sql WHERE $col_sql REGEXP '^[aiO]:[1-9]' LIMIT 1" );
```

Comment:
As there is now a comment breaking the assignment list, the whitespace before the `=` here needs to be reduced to one space.